src/EnergyPlus/SetPointManager.cc

```diff
@@ -250,9 +278,6 @@ void GetSetPointManagerInputData(EnergyPlusData &state, bool &ErrorsFound)
     using ScheduleManager::GetScheduleIndex;
 
     // Locals
-    // SUBROUTINE PARAMETER DEFINITIONS:
-    const char *RoutineName("GetSetPointManagerInputs: "); // include trailing blank space
-
```

Comment:
We deleted this so that we don't need to use std::string{}, as recommended by Amir. Then, the format function is used, instead.